es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
+ 
 <br>
 
 **7. [Metric, Formula, Interpretation]**
 
-&#10230;
+&#10230; [Métrica, Fórmula, Interpretación]*
 
 <br>
 
 **8. Overall performance of model**
 
-&#10230;
+&#10230; Desempeño en general del modelo
 
 <br>
 
 **9. How accurate the positive predictions are**
 
-&#10230;
+&#10230; ¿Qué tan precisan son las precisiones positivas?
 
 <br>
 
 **10. Coverage of actual positive sample**
 
-&#10230;
+&#10230; Covertura de las pruebas positivas
 
 <br>
 
 **11. Coverage of actual negative sample**
 
-&#10230;
+&#10230; Covertura de las pruebas negativas
 
 <br>
 
 **12. Hybrid metric useful for unbalanced classes**
 
-&#10230;
+&#10230; Métrica híbrida útil para clases no balanceadas
 
 <br>
 
 **13. ROC ― The receiver operating curve, also noted ROC, is the plot of TPR versus FPR by varying the threshold. These metrics are are summed up in the table below:**
 
-&#10230;
+&#10230; Curva Curva de operación del receptor (ROC) ― La curva receptora de operaciones, por sus siglás en inglés ROC, es la gráfica de los Tasa de verdaderos positivos" (TPR) contra los Tasa de falsos positivos (FPR) variando el umbral. Estas métricas son resumidas en la tabla siguiente: 
 
 <br>
 
 **14. [Metric, Formula, Equivalent]**
 
-&#10230;
+&#10230; [Métrica, Fórmula, Equivalente]
 
 <br>
 
 **15. AUC ― The area under the receiving operating curve, also noted AUC or AUROC, is the area below the ROC as shown in the following figure:**
 
-&#10230;
+&#10230; AUC ― Área bajo la curva ROC, también llamda AUC o AUROC, es la área bajo de la curva ROC como se muestra en la siguiente figura:
 
 <br>
 
 **16. [Actual, Predicted]**
 
-&#10230;
+&#10230; [Valor actual, Predicho]
 
 <br>
 
 **17. Basic metrics ― Given a regression model f, the following metrics are commonly used to assess the performance of the model:**
 
-&#10230;
+&#10230; Métricas básicas ― Dado un modelo de regresión f, las siguientes métricas son comunmente usadas para evaluar el desempeño del modelo:
 
 <br>
 
 **18. [Total sum of squares, Explained sum of squares, Residual sum of squares]**
 
-&#10230;
+&#10230; [Suma total de los cuadrados, Explicaicón de la suma de cuadrados, Residio de la suma de cuadrados]
 
-<br>
+<br> 
 
 **19. Coefficient of determination ― The coefficient of determination, often noted R2 or r2, provides a measure of how well the observed outcomes are replicated by the model and is defined as follows:**
 
-&#10230;
+&#10230; Coefficientes de determinante ― El coeficiente de determinante, usualmente llamado R2 o r2, provee una medida 
 
 <br>
 
 **20. Main metrics ― The following metrics are commonly used to assess the performance of regression models, by taking into account the number of variables n that they take into consideration:**
 
-&#10230;
+&#10230; Métricas principales ― Las siguientes métricas son comunmente usadas para evaluar el desempeño de los modelos de regresión, tomando el número de variables n que son consideradas:
```

Comment:
* _Comúnmente_
* "taking into account" is literally translated as _teniendo en cuenta_.